Description: ignore Freerunner's GPS by default
 ignore the GPS device of Openmoko GTA01/GTA02
Author: Rico Rommel <rico@bierrommel.de>
--- a/conf/openmoko_gta/fsodeviced.conf
+++ b/conf/openmoko_gta/fsodeviced.conf
@@ -66,3 +66,4 @@
 ignore_wifi = true
 ignore_bluetooth = true
 ignore_usbhost = false
+ignore_gps = true
